public/components/freelancer-create.js

```diff
@@ -81,14 +81,16 @@ class CreationForm extends React.Component {
               </label>
             </div>
 
-            <label className="category-selector">
-              Job category:
+            <div className="category-selector">
+              <span className="bar"></span>
+              <label>
+                Job Category
+              </label>
               <select ref="category">
                 <option value="selected" disabled>Please select a job category</option>
                 {this.props.categories}
               </select>
-
-            </label>
+            </div>
 
             <div className="group">
               <input ref="address" className="address" name="address" type="text" required/>
```

Comment:
Should not be required, as in the model. The only required fields should be `name`, `title`, `category`, `email`.